pretix_sepadebit/migrations/sepaduedate.py

```diff
@@ -0,0 +1,64 @@
+# Generated by Django 3.0.14 on 2021-04-22 20:47
```

Comment:
The name of the migration should start with ``0007_`` to keep order in the migrations directory (Django usually does that automatically)